README.md

```diff
@@ -55,10 +55,10 @@ Therefore it should be used with explicit dependencies:
 
 ### Adding a new module to the stack
 
-1. Add it to `vertx-dependencies` (open the project, add it to the `pom.xml`, install, commit and push)
+1. Add it to [`vertx-dependencies`](https://github.com/vert-x3/vertx-dependencies) (open the project, add it to the `pom.xml`, install, commit and push)
 2. Add it to `stack-depchain/pom.xml` (open the project, add the dependency, without the version (inherited from
 vertx-dependencies))
-3. Add it in the stack manager:
+3. Add it in the stack manager: **Only required for certain modules**
```

Comment:
To my knowledge, this is required for all modules, which one do you have in mind?

Reply:
Don't know exactly but Julien said it is for "dependencies convergence" and I don't need to add the module there.

@vietj could you clarify this?

Reply:
I removed this part.

Reply:
Thanks!